features/step_definitions/product_listing_steps.rb

```diff
@@ -10,12 +10,4 @@
     table.hashes.each do |category|
       Category.create!(category)
```

Comment:
Can we change this to the factory version? eg ```create(:category)``` instead of ```Category.create!(category)```

Reply:
removed Factory on line 5, which was redundant. But don't get your proposal to work for the test, so keeps it as is for now.

Reply:
Sorry my bad - use ```create(:category, category)``` that should pass